docs/breaking-changes.md

```diff
@@ -38,6 +38,11 @@ The autoresizing behavior is now standardized across all platforms.
 If your app uses `BrowserView.setAutoResize` to do anything more complex than making a BrowserView fill the entire window, it's likely you already had custom logic in place to handle this difference in behavior on macOS.
 If so, that logic will no longer be needed in Electron 30 as autoresizing behavior is consistent.
 
+### Deprecated: `inputFieldType` in `context-menu` params
+
+The `inputFieldType` property in the `context-menu` params has been deprecated and will be removed.
+Use `formControlType` instead.
+
```

Comment:
Can we add a little more info about how previous usage should be changed? A before/after example would be ideal.